parts/k8s/cloud-init/artifacts/apt-preferences

```diff
@@ -0,0 +1,3 @@
+Package: walinuxagent
+Pin: version 2.2.32.2
```

Comment:
(not in this PR but side note -) we should revisit this. I don't think it's doing anything that we want it to do.

Reply:
yep